src/external/views/nunjucks/service-status/index.njk

```diff
@@ -1,68 +1,67 @@
-<html>
-   <head>
-   <body>
+{% extends "./nunjucks/layout.njk" %}
```

Comment:
Should we now hide all the stats for external users and just show 'OK'?

Reply:
I think we agreed that this desicion would be outside of this story.